src/tests/cli-tests/src/helper.ts

```diff
@@ -0,0 +1,29 @@
+import * as shell from 'shelljs';
+import * as fs from 'fs';
+
+interface CommandResult {
+    output: string;
+    exitCode: number;
+}
+
+export const executeCommand = (command: string): CommandResult  => {
+    const result  = shell.exec(command, {async: false});
+    return {
+        exitCode: result.code,
+        output: result.stdout.trim() || result.stderr.trim(),
```

Comment:
It is worth to test trimming between `solc` and `zksolc` by the way.
I've never checked how `solc` performs it.

Reply:
Good point, I have added a subtask for it "QA-619" to research the question. (not sure if we can copy a full link to our tracker here)